source/community/groups/auth-tsg/Charter.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "IIIF Authorization Charter"
+layout: spec
+tags: []
+cssversion: 2
+---
+## Introduction
+The IIIF Authentication API is an interaction pattern for managing access control to IIIF image resources. With the introduction of support for A/V in the IIIF Presentation API version 3 there is now a need to support authenticating resources which do not require a IIIF Image API service, as well as general updates for consistency with the new APIs. There have also been developments in the browser community that cause difficulties with the existing authentication specification. With the current specification being broken in some browsers, and soon to be obsolete in others, there is an urgent need to update the method in which the IIIF Authentication API works to remain useful in the evolving web landscape.
+
+## Scope
+The scope of work to be performed by the Authentication TSG will be:   
+* Updating the IIIF Authentication API in line with version 3.0 of the IIIF Image and Presentation APIs
+* Finding and documenting a solution to the browser issues that have arisen, with an initial approach of engaging with the standards and browser communities
+* Determining a IIIF Authentication API pattern for access to non-Image content resources
+* Explore Authentication patterns for access to resources requested through JavaScript without exposing authorization credentials to the client (such as for searching annotations with credentials)   
+
+## Deliverables
+The expected deliverable will be a new version of the IIIF Authentication API that meets the requirements of the scope described above.   
+
+## References
+References for implementations of the third-party cookie changes:
+* Safari issue
+* Chrome plans
+* IIIF API issue: https://github.com/IIIF/api/issues/1959   
+
+Stories that support this TSG work:
+* Canvas with content that will never be "seen"
+* And the following issues that have been raised on the API Github repository:
+https://github.com/IIIF/api/issues?q=is%3Aissue+is%3Aopen+label%3Aauth   
+
+## Roadmap
+* Group formation: October 2020
+* Delivery of new Authentication API version, given current technical /political environment: Initial draft: June 2021; Implementable Beta: November 2021; Proposed Recommendation: June 2022   
+
+## Community Support
+Please add your name and institution below to show your support to the formation of the group:   
```

Comment:
I'm not sure we should include the names of people in the community support part of the charter. We haven't done so previously and its probably not a good idea without their permission.